src/mixxx.cpp

```diff
@@ -252,12 +250,19 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
         exit(-1);
     }
 
+    ChannelHandleFactoryPtr pChannelHandleFactory =
```

Comment:
can use `auto pChannel[...] =` because the type is trivially deduced from the right hand side of the assignment